MDANSE_GUI/Src/MDANSE_GUI/InputWidgets/MoleculePreviewWidget.py

```diff
@@ -0,0 +1,155 @@
+from qtpy.QtWidgets import QWidget, QSizePolicy, QVBoxLayout, QLabel, QDialog
+from qtpy.Qt3DExtras import Qt3DWindow
```

Comment:
We'll need to update the pyproject.toml and add pyqt3d as a dependency. @MBartkowiakSTFC I guess we should add PyQt6-3D since PyQt6<=6.7.0 is in the pyproject.toml.